misc/object-conditions/basics.rst

```diff
@@ -392,6 +392,14 @@ Available operators for matching:
    Triggers, Time Accounting selector, Postmaster Filters, Automatic ticket
    assignment and Core Workflow.
 
+.. hint::
+   **What about my "old" style regex filters?**
```

Comment:
Perhaps we should switch `regex` to something like

```
``regex:...``
```

So it's clear to the readers which old style are you refering to.